spec/ttml1.xml

```diff
@@ -13406,7 +13405,7 @@ during playback, presentation timing is not affected.
 <p>When operating with the <code>media</code> time base, the following semantics should be applied for interpreting time expressions,
 as defined by <loc href="#timing-value-timeExpression">&lt;timeExpression&gt;</loc>, and their relationship to media time and local real time.</p>
 <p>The media time base <phrase role="strong"><code>M</code></phrase> is related to local real time <phrase role="strong"><code>R</code></phrase>
-expressed in the applicable epoch <phrase role="strong"><code>E</code></phrase> as follows:</p>
+expressed in the epoch <phrase role="strong"><code>E</code></phrase> (defined by the <emph>Document Processing Context</emph>) as follows:</p>
```

Comment:
leave "applicable"; remove parenthesis; change to "as defined by ..."

Reply:
"as defined by" will be followed by "as follows", which I think reads funny.

Reply:
I thought you were going to use @cconcolato's text, which would be:

```
expressed in the epoch <phrase role="strong"><code>E</code></phrase> defined by the Document Processing Context as follows:
```

which is fine with me.

Reply:
I think you're both wrong but also right! The point is not to give the impression that the "as follows" explains how the document processing context defines the epoch. How about:

_expressed in the applicable epoch E, where E is defined by the Document Processing Context, as follows:_

note the use of a comma after Document Processing Context to separate the sub-clause.

Reply:
I added the parentheses to emphasize that "as follows" qualifies epoch E and not _Document Processing Context_

Reply:
I think proper parentheses are clearer here than commas. I'm still worried that the comma separated sub-clause could be misread.